op-chain-ops/genesis/setters.go

```diff
@@ -60,6 +78,37 @@ func SetL1Proxies(db vm.StateDB, proxyAdminAddr common.Address) error {
 	return setProxies(db, proxyAdminAddr, bigL1PredeployNamespace, 2048)
 }
 
+// WipeStorage will set every storage variable that is set to `bytes32(0)`.
+func WipeStorage(db vm.StateDB, addr common.Address) error {
+	err := db.ForEachStorage(addr, func(key, _ common.Hash) bool {
```

Comment:
In the interest of speed, let's do `db.CreateAccount` instead and call the function WipePredeploy

Reply:
Updated to this suggestion